R/iSEEinit.R

```diff
@@ -79,7 +90,7 @@ iSEEinit <- function(sce,
             paste(not_available_features, collapse = ", "))
 
     features <- features[features %in% rownames(sce)]
-    if (length(features) == 0)
+    if (NROW(features) == 0)
```

Comment:
OK, TIL that `NROW` exists...

Reply:
yup, it's more flexible than `length` if we want to introduce `data.frame` 🤓